test/integration/compute/serverspec/compute_spec.rb

```diff
@@ -76,3 +76,7 @@
   it { should be_owned_by 'nova' }
   it { should be_grouped_into 'nova' }
 end
+
+describe file('/etc/nova/nova.conf') do
+  its(:content) { should match(/^disk_cachemodes = file=writeback,block=writeback$/) }
+end
```

Comment:
I'd instead of testing this here, I recommend you update the ChefSpec and just remove this please. We don't need to replicate checking of this in ServerSpec when we can do it much faster in ChefSpec.